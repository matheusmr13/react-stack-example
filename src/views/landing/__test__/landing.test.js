import React from 'react';
import { shallow } from 'enzyme';
import Landing from 'views/landing';

describe('Landing component', () => {
	process.env.REACT_APP_SPOTIFY_AUTH_URL = 'spotify';
	process.env.REACT_APP_SPOTIFY_CLIENT_ID = '123';
	it('should render properly', () => {
		const landing = shallow(<Landing />);
		expect(landing).toMatchSnapshot();
	});
});
