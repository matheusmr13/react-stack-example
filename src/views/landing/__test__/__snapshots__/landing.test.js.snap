// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing component should render properly 1`] = `
<div
  className="landing"
>
  <section>
    <div
      className="fade"
    />
    <div
      className="content content--centralized"
    >
      <div
        className="centralized"
      >
        <div
          className="landing__logos"
        >
          <a
            href="https://www.spotify.com/"
          >
            <img
              alt="Spotify"
              src="spotify-white-logo.png"
            />
          </a>
          <span>
            +
          </span>
          <a
            href="https://www.ifood.com.br/"
          >
            <img
              alt="IFood"
              src="ifood-white-logo.png"
            />
          </a>
        </div>
        <div
          className="landing__disclaimer"
        >
          <KeyboardSimulator
            text="Welcome to a new"
            values={
              Array [
                "Experience",
                "Thing",
                "Life",
              ]
            }
          />
        </div>
        <a
          className="landing__login"
          href="spotify?client_id=123&response_type=token&redirect_uri=http://localhost/"
        >
          Enter with Spotify
        </a>
      </div>
    </div>
  </section>
  <footer
    className="landing__footer"
  >
    <div>
      Made by matheusmr13
    </div>
    <div>
      <a
        href="https://github.com/matheusmr13"
      >
        <img
          alt="matheusmr13"
          src="github-icon.svg"
        />
      </a>
      <a
        href="https://www.linkedin.com/in/matheus-martins-do-rego-9593a484/"
      >
        <img
          alt="Matheus Martins"
          src="linkedin-icon.svg"
        />
      </a>
    </div>
  </footer>
</div>
`;
